weekly-mission(week2)/src/constants/errorMessage.js

```diff
@@ -0,0 +1,18 @@
+const ERROR_MESSAGE = Object.freeze({
+  EMAIL: {
+    BLANK: '이메일을 입력해 주세요.',
+    INVALID: '올바른 이메일 주소가 아닙니다.',
+    INCORRECT: '이메일을 확인해 주세요.',
+    DUPLICATE: '이미 사용 중인 이메일입니다.',
+  },
+  PASSWORD: {
+    BLANK: '비밀번호를 입력해 주세요.',
```

Comment:
요기도 EMPTY가 더 좋을 것 같습니다!
```suggestion
    EMPTY: '비밀번호를 입력해 주세요.',
```